### ISSUE TITLE
Do Polls Work?

### ISSUE THREAD

**Issue by KevinDuganJr** [YAFNET/YAFNET#953]
Maybe this was addressed in one of the updates. I couldn't find anything, so thought I'd ask. Someone created a Poll and it says I already voted, yet no votes were made. I also have not voted.
![image](https://github.com/YAFNET/YAFNET/assets/5612416/6f961d6d-8d5c-4427-8926-227acd8a2abf)

My Version.
![image](https://github.com/YAFNET/YAFNET/assets/5612416/5dd420a8-34fc-4e96-bc57-c6551da861e7)

**Comment by w8tcha** [comment]
Yes this issue was definitely fixed in on of the latest version 3.1.1x

**Comment by KevinDuganJr** [comment]
> Yes this issue was definitely fixed in on of the latest version 3.1.1x

I am not trying to pressure you or be an impatient jerk here, but before I go through updating my site. Do you about when the newest YAF will be released with the YAF Editor returning? 😬😬😬

**Comment by w8tcha** [comment]
The new (old) improved editor is back in the code base. New release should follow in the next few days

**Comment by KevinDuganJr** [comment]
> The new (old) improved editor is back in the code base. New release should follow in the next few days

Awesome. I am going to see if I have better luck upgrading now. Crossing fingers!! haha
